src/cell.cpp

```diff
@@ -252,12 +252,12 @@ void Cell::to_hdf5(hid_t cell_group) const
 // default constructor
 CSGCell::CSGCell()
 {
-  geom_type_ = GeometryType::CSG;
+  geom_type() = GeometryType::CSG;
```

Comment:
There are a couple usages of `geom_type_` directly in `particle.cpp` (and perhaps elsewhere). Can you update those lines as well please?

Reply:
changed all `geom_type_` call to `geom_type()`
change `suface.h` accordingly